src/Filter/Sprockets.php

```diff
@@ -139,21 +139,23 @@ public function getDependencies(AssetTarget $target)
         $children = [];
         foreach ($target->files() as $file) {
             $contents = $file->contents();
-            preg_match($this->_pattern, $contents, $matches);
+            $count = preg_match_all($this->_pattern, $contents, $matches);
             if (empty($matches)) {
                 continue;
             }
 
-            if ($matches[1] === '"') {
-                // Same directory include
-                $path = $this->_findFile($matches[2], dirname($file->path()) . DIRECTORY_SEPARATOR);
-            } else {
-                // scan all paths
-                $path = $this->_findFile($matches[2]);
+            for ($i = 0; $i < $count; $i ++) {
+                if ($matches[1][$i] === '"') {
+                    // Same directory include
+                    $path = $this->_findFile($matches[2][$i], dirname($file->path()) . DIRECTORY_SEPARATOR);
+                } else {
+                    // scan all paths
+                    $path = $this->_findFile($matches[2][$i]);
+                }
+                $dep = new Local($path);
+                $children[] = $dep;
```

Comment:
Thanks for the info on how to reproduce the problem . I think I've addressed this issue and the scss dependency issue in #19.